extensions/quarkus-rest-jackson/runtime/src/main/resources/quarkus-extension.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: "Quarkus REST JSON-B"
```

Comment:
```suggestion
name: "Quarkus REST Jackson"
```

Reply:
:+1: . I'll change it when I go through the commit history and try to clean it up